src/collator/collator.py

```diff
@@ -67,13 +66,13 @@ def __init__(
 
         # store feedback as flattened array. if no feedback provided, use empty strings
         self.feedback = (
-            np.hstack(feedback)
-            if feedback is not None
+            np.hstack(custom_dataset.feedback)
+            if feedback
```

Comment:
Instead of using a random vector for the "empty" feedback. Why don't we generate the embedding for a fixed sentence like "no feedback available"? In this way, we avoid using a representation that might diverge too much from the original embedding space learned by sentence-bert.

Reply:
Good point!